estela-web/src/components/ProjectListPage/index.tsx

```diff
@@ -171,18 +180,60 @@ export class ProjectListPage extends Component<unknown, ProjectsPageState> {
             count: data.count,
             current: data.current,
             loaded: true,
+            modalWelcome: data.count === 0,
         });
     };
 
     render(): JSX.Element {
-        const { projects, count, current, loaded, modalNewProject, newProjectName, newProjectCategory } = this.state;
+        const { projects, count, current, loaded, modalNewProject, modalWelcome, newProjectName, newProjectCategory } =
+            this.state;
         return (
             <Layout className="h-screen">
                 <Header />
                 <Layout className="bg-metal p-6">
                     {loaded ? (
                         <Fragment>
                             <Content className="mx-4">
+                                <Modal
+                                    open={modalWelcome}
+                                    footer={false}
+                                    width={990}
+                                    onCancel={() => {
+                                        this.setState({ modalWelcome: false });
+                                    }}
+                                >
+                                    <Row className="py-8 px-4" align="middle">
+                                        <Col span={16}>
+                                            <Text className="text-estela font-bold text-4xl">WELCOME SCRAPER!</Text>
+                                            <Paragraph className="text-xl mt-6">
+                                                Start by creating a <Text strong>project</Text> to be able to deploy
+                                                your spiders and start with your scraping.
+                                            </Paragraph>
+                                            <Paragraph className="text-lg font-bold">
+                                                Remember to install the&nbsp;
+                                                <a
+                                                    target="_blank"
+                                                    href="https://estela.bitmaker.la/docs/"
```

Comment:
I think the href is **https://estela.bitmaker.la/docs/estela-cli/install.html**

Reply:
updated.